packages/base-styles/_variables.scss

```diff
@@ -91,7 +102,6 @@ $border-width: 1px;
 $border-width-focus-fallback: 2px; // This exists as a fallback, and is ideally overridden by var(--wp-admin-border-width-focus) unless in some SASS math cases.
 $border-width-tab: 1.5px;
 $helptext-font-size: 12px;
-$radius-round: 50%;
 $radius-block-ui: 2px;
```

Comment:
Should `$radius-s` replace the `$radius-block-ui` or be part of its declaration?

Reply:
I'd welcome thoughts on that. My idea was to essentially deprecate `$radius-block-ui` by assigning it the `radius-s` value like you suggest, then updating all the styles that utilises it. That second step is a big one though.

Reply:
> becomes even more confusing to have in the first place

I agree, but I assume it would not be safe to remove?

Reply:
The safest thing would be to keep it around, aliasing it to `$radius-small`. We could then also add a deprecation notice (I don't know how those work for Sass variables), and maybe a stylelint rule to prevent further usages.

Not sure if we'd be able to remove it entirely, since a missing Sass variable would cause build errors. So, on one end it would be easy to spot and fix (especially if we add a dev note), on the other end I'm not sure we're ok with causing build errors.

Reply:
Sounds like a plan :)

Reply:
I pushed a commit to update/deprecate `$block-radius-ui`: https://github.com/WordPress/gutenberg/pull/64007/commits/ea32b3345694ff445eb935b4482deea81c862730. I don't mind doing that in a separate PR if y'all prefer though.